pages/common/etcdctl.md

```diff
@@ -0,0 +1,37 @@
+# etcdctl
+
+> CLI interface for interacting with etcd high performance key/value pair store.
+> Etcd stores keys in a filesystem-like format, e.g. /my/key.
+> More information: <https://etcd.io/docs/latest/dev-guide/interacting_v3/>.
+
+- Get a key:
+
+`etcdctl get {{/my/key}}`
```

Comment:
Why the key with `/`?

Reply:
@navarroaxel You mean the prefix?  That's what etcd does...see the homepage (https://etcd.io).

>### Key-value storage
>
>Store data in hierarchically organized directories, as in a standard filesystem
>```
>/config
>  /database
>/feature-flags
>  /verbose-logging
>  /redesign
>```

Reply:
Oh, OK! but the `/` is required? The examples in more information link uses: `Here is the command to set the value of key foo to bar` without any `/`.

Reply:
Good point @navarroaxel.  I suppose we can remove it.

Reply:
@navarroaxel done @ 3ceea82781a8d89520ceaf24720a208e0c3483af

Reply:
Yeah, I think that will be better.